stylesheets/main.scss

```diff
@@ -830,6 +830,10 @@ html:not(.f-coarse-pointer) body nav > .ui.segment .f-nav-view:not(.dropzone) .f
     width: 50ch;
 }
 
+.f-sender {
+    float: right;
```

Comment:
If we can avoid float we should , it's dangerous.  Probably can use a flex container with two divs inside it that are given appropriate weights.